### ISSUE TITLE
[Apache] prepare English version of user documentation

### ISSUE THREAD

**Issue by javeme** [hugegraph/hugegraph#1645]
### Feature Description (功能描述)

At present, the user documentation is in Chinese, and an English version needs to be prepared, and issues also need to guide users to use English.

**Comment by javeme** [comment]
related issue: #142

**Comment by lzm0** [comment]
I agree that people should be encouraged to use English when contributing to this project. A long conversation in Chinese does not help developers globally.
Apache also addressed the importance of diversity to the community. https://incubator.apache.org/guides/graduation.html#creating_an_open_and_diverse_community

**Comment by imbajin** [comment]
Done in: https://github.com/apache/hugegraph-doc/tree/master/content/en
